docs/source/tutorials/vision/plot_simple_classification_tutorial.py

```diff
@@ -51,8 +59,8 @@
 
 from deepchecks.vision.simple_classification_data import load_dataset
 
-train_ds = load_dataset('hymenoptera_data', train=True, object_type='VisionData', image_extension='jpg')
-test_ds = load_dataset('hymenoptera_data', train=False, object_type='VisionData', image_extension='jpg')
+train_ds = load_dataset('./EuroSAT/euroSAT/', train=True, object_type='VisionData', image_extension='jpg')
+test_ds = load_dataset('./EuroSAT/euroSAT/', train=False, object_type='VisionData', image_extension='jpg')
 
 # Running Deepchecks' full suite
```

Comment:
These lines would be better as an external title and explanation and not as comments inside the code snippet